src/guide/lists.md

```diff
@@ -14,7 +14,7 @@ let numbers = [1, 2, 3]
 let strings = ["foo", "bar", "baz"]
 ```
 
-Lists in Grain are linked lists, so if we'd like to add a new item to a list, we add it to the front:
+Lists in Grain are linked lists, so if we'd like to add a new item to a list, the preferred way is to add it to the front, which we can do with the spread syntax (`...`):
```

Comment:
I might even stick to just saying that the way you do it is by adding it to the front, to get them in the mindset of "this is how I'm supposed to do it" and later introduce that it can be done another way.